src/index.ts

```diff
@@ -45,12 +46,12 @@ export class TxWithdraw {
   opcode: BN;
   l1address: Address;
 
-  constructor(nonce:BN, accountIndex:BN, tokenIndex:BN, amount:BN, l1address: Address) {
+  constructor(nonce:BN, accountIndex:BN, tokenIndex:BN, amount:BN, l1address: Address, networkId: string) {
```

Comment:
input of networkid must be int. This will force user to input correct decimal number.